fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -645,4 +725,25 @@ private void checkLinkForLdpResourceCreation(final String link) {
         }
     }
 
+    /**
+     * Parse the RFC-3230 Digest response header value.  Look for a
+     * sha1 checksum and return it as a urn, if missing an empty string is
+     * returned.
+     * @param digest The Digest header value
+     * @return the sha1 checksum value
+     */
+    private String parseDigestHeader(final String digest) {
+        if (digest != null) {
+            final Map<String,String> digestPairs = RFC3230_SPLITTER.split(digest);
+            for (Map.Entry<String, String> digestPair : digestPairs.entrySet()) {
+                switch(digestPair.getKey().toLowerCase()) {
```

Comment:
Before deciding on how or if the `parseDigestHeader` signature should change, what is the expected behavior of this PR (noting that there is another [ticket](https://jira.duraspace.org/browse/FCREPO-1842) to allow for alternate checksums)?
To be more specific, what do we expect the response to be for the following client requests:
1. POST -H"Digest: md5=some-md5" /resource
2. POST -H"Digest: md5=some-md5,SHA1=some-sha1" /resource
3. POST -H"Digest: sha1=some-sha1" /resource